calico/felix/devices.py

```diff
@@ -54,6 +56,11 @@ def check_kernel_config():
                       "0 or 1.")
         raise BadKernelConfig("net.ipv4.conf.all.rp_filter set to 'loose'")
 
+    # Make sure the default for new interfaces is set to strict checking so
```

Comment:
I think the log message is now a bit confusing.  It needs to be 1; just tell them to set it to 1.

I'm also a bit unsure about allowing setting 2 through.  Let's chat.